src/stores/EventDetailsStore.js

```diff
@@ -42,8 +48,18 @@ class EventDetailsStore extends EventEmitter {
     setDefaultEventDetails() {
         this.eventDetail.eventDetailVisible = false;
         this.eventDetail.events = [];
+        this.subscribed = false;
         return this.eventDetail;
     }
+
+    isUserSubscribed(isSubscribed){
```

Comment:
Eu não entendi a nomenclatura do `is` no nome da função, porque para mim isto é uma pergunta, só que aqui no `Store` é a execução da ação então não teria um sim ou não, seria algo do tipo, `subscribeUserToEvent`.